D1673/P1673.md

```diff
@@ -8550,11 +8531,15 @@ void triangular_matrix_vector_solve(
   BinaryDivideOp divide);
 ```
 
-[1]{.pnum} *Effects:* Computes $x'$ such that $b = A x'$,
+[5]{.pnum} This function performs a triangular solve taking into account the `Triangle` and `DiagonalStorage` parameters [linalg.general].
+
+[6]{.pnum} *Effects:* Computes $x'$ such that $b = A x'$,
 and assigns each element of $x'$ to the corresponding element of `x`.
 If no such $x'$ exists,
 then the elements of `x` are valid but unspecified.
 
+[7]{.pnum} *Complexity:* $O($ `x.extent(0)` $*$ `A.extent(1)` $)$
```

Comment:
I would have spelled this

$O($ `A.extent(0)` $*$ `x.extent(0)` $)$.

Even though A is square, using the number of rows of A gives the right "units."